src/compiler/parser.ts

```diff
@@ -7093,13 +7093,20 @@ namespace Parser {
                 case SyntaxKind.PrivateKeyword:
                 case SyntaxKind.ProtectedKeyword:
                 case SyntaxKind.PublicKeyword:
-                case SyntaxKind.ReadonlyKeyword:
+                case SyntaxKind.ReadonlyKeyword: {

```

Comment:
Nit: I know it helps with scope hygiene, but since the rest of the codebase avoids it, I'd rather we didn't add braces unless necessary.

```suggestion
                case SyntaxKind.ReadonlyKeyword:
```